cassandra-diagnostics-connector21/README.md

```diff
@@ -0,0 +1,51 @@
+# Cassandra Diagnostics Connector for Cassandra 2.1.x
+
+Connector is a module which hooks into the query path and extract information for diagnostics. Bytecode instrumentation is used to augment existing Cassandra code with addition functionality. It uses low priority threads to execute the diagnostics information extraction with minimal performance impact to the target code (Cassandra node or application/driver).
+
+## Tracing Slow Queries
```

Comment:
Can you explain tracing on example of slow queries, not exclusive and connected with only slow queries.

Imagine tomorrow we have number of rows returned module which measures how many rows are returned per select (checking if select is returning too much). Tracing in this module will give statement which is problematic. So tracing is not connected to only slow queries, and that is reason why it is configured per connector and not per slow queries.